### ISSUE TITLE
[BUG] Calling the integer resolution HTTP endpoint results in a numeric string value

### ISSUE THREAD

**Issue by toddbaert** [open-feature/flagd#125]
### Observed behavior

Calling the integer resolution endpoint results in a string value (ie: `"1"`).

### Expected Behavior

Calling the integer resolution endpoint results in integer JSON value (ie: `1`).

### Steps to reproduce

Start flagd:

```
./flagd start -f config/samples/example_flags.json --service-provider http --sync-provider filepath
```

Call the documented integer method:

```
$ curl -X POST "localhost:8013/flags/myIntFlag/resolve/int"
{"value":"1", "reason":"STATIC", "variant":"one"}
```

@beeme1mr noticed this.I think the response serializer is treating the int64 as a char for some reason. Not exactly sure why...

**Comment by james-milligan** [comment]
[behaviour introduced in this PR](https://github.com/open-feature/flagd/pull/99)

when using int64 the http interface will return a string value, this is due to the [proto3 json mapping](https://developers.google.com/protocol-buffers/docs/proto3#json) within grpc-gateway.

this behaviour is not present in the grpc implementation

**Comment by toddbaert** [comment]
I think we can close this with some minor doc, as discussed.

**Comment by james-milligan** [comment]
ill sort this now

**Comment by toddbaert** [comment]
Closed with #132
